src/screens/ProfileScreen/index.js

```diff
@@ -723,7 +722,7 @@ const ProfileScreen = ({route}) => {
         onClose={onHeaderOptionClosed}
         onDeleteClicked={onDeletePost}
       />
-    </SafeAreaProvider>
+    </View>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The closing tag for the root container has been changed from `SafeAreaProvider` to `View`. This change should correspond with the opening tag.
```diff
-</SafeAreaProvider>
+</View>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->